pkg/collector/corechecks/sbom/processor.go

```diff
@@ -316,11 +324,12 @@ func (p *processor) processImageSBOM(img *workloadmeta.ContainerImageMetadata) {
 		}
 
 		sbom := &model.SBOMEntity{
```

Comment:
Do you mean that we should check if we have a repo digests for the considered image. And, if we don’t, we log an error instead of sending the SBOM payload ?

Reply:
It depends. For error cases I assume that we might want to retrieve the error even if we can't associate it to its SBOM. For the pending and successful cases I don't see why we would need to send it.

IMO it could be useful to keep sending it in case we want to fallback on the `image_id`